### PR TITLE
[Documentation/Shapes] & [BottomSheet]  markdown fixes for code preview errors

### PR DESCRIPTION
**Pull request by claytonmeador**
#### Don't forget:
- [ ] Identify the component the PR relates to in brackets in the title. ```[Buttons] Updated documentation```
- [ ] Link to GitHub issues it solves. ```closes #1234```
- [ ] Sign the CLA bot. You can do this once the pull request is opened.

[Contributing](./contributing/README.md#pull-requests) has more information and tips for a great
pull request.

### CODE REVIEW

File: components/BottomSheet/README.md [hunk 1]
```diff
@@ -99,7 +102,5 @@ button.addTarget(self, action: #selector(presentBottomSheet), for: .touchUpInsid
 ```objc
 _button = [[UIButton alloc] initWithFrame:CGRectZero];
 [_button addTarget:self action:@selector(presentBottomSheet) forControlEvents:UIControlEventTouchUpInside];
-
 ```
-
 <!--</div>-->
```
Comment (jverkoey) [inline components/BottomSheet/README.md:106]: Please make the same changes to `components/BottomSheet/docs/` (the source content for this file).

Comment (claytonmeador) [reply components/BottomSheet/README.md:106]: Will do sorry about that!

Comment (jverkoey) [reply components/BottomSheet/README.md:106]: Gentle ping :)

**Comment by romoore** [comment]
Closing this due to inactivity. Please feel free to reopen if you find time to work on it.

**Pull request closed without merging**